lab-anthony/controller/user-controller.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
```

Comment:
It seems goofy to have a user controller and an auth controller, which handles user creation and lookup.